platform/src/test/kotlin/org/stellar/anchor/platform/service/Sep31DepositInfoGeneratorApiTest.kt

```diff
@@ -0,0 +1,85 @@
+package org.stellar.anchor.platform.service
+
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import java.util.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.ValueSource
+import org.stellar.anchor.api.callback.GetUniqueAddressResponse
+import org.stellar.anchor.api.callback.UniqueAddressIntegration
+import org.stellar.anchor.api.exception.HttpException
+import org.stellar.anchor.sep31.Sep31Transaction
+
+class Sep31DepositInfoGeneratorApiTest {
+  @MockK(relaxed = true) private lateinit var uniqueAddressIntegration: UniqueAddressIntegration
+  @MockK(relaxed = true) private lateinit var txn: Sep31Transaction
+  val txnId = "this_is_a_transaction_id"
+  val stellarAddress = "GBJDSMTMG4YBP27ZILV665XBISBBNRP62YB7WZA2IQX2HIPK7ABLF4C2"
+
+  @BeforeEach
+  fun setup() {
+    MockKAnnotations.init(this, relaxUnitFun = true)
+  }
```

Comment:
❗ You forgot to call `@AfterEach` and unmock everything. If you don't do it, the mocks will remain.

Reply:
Fixed.